Vagrantfile

```diff
@@ -14,6 +14,9 @@ else
   raise "#{config_file} file not found. Please set `ANSIBLE_PATH` in Vagrantfile"
 end
 
+# Set Ansible roles_path relative to Ansible directory
+ENV['ANSIBLE_ROLES_PATH'] = File.join(ANSIBLE_PATH, 'vendor/roles')
```

Comment:
Could you move this up under `ANSIBLE_PATH`? Rather just have them grouped.

Also: `File.join(ANSIBLE_PATH, 'vendor', 'roles')`